vpt/processors/gaze_detection/__init__.py

```diff
@@ -0,0 +1,226 @@
+"""Human facial landmark detector based on Convolutional Neural Network."""
+import cv2
+import numpy as np
+import tensorflow as tf
+from rx import Observable
+from rx.subject import Subject
+from tensorflow import keras
+import math
+
+# import debug
+from data_structures import VideoFrame
+from nodes import ProcessorBase, SourceBase
+
+
+class FaceDetector:
+    """Detect human face from image"""
+
+    def __init__(self,
+                 dnn_proto_text='models/deploy.prototxt',
+                 dnn_model='models/res10_300x300_ssd_iter_140000.caffemodel'):
+        """Initialization"""
+        self.face_net = cv2.dnn.readNetFromCaffe(dnn_proto_text, dnn_model)
+        self.detection_result = None
+
+    def get_faceboxes(self, image, threshold=0.5):
```

Comment:
```suggestion
    def get_faceboxes(self, image: np.ndarray, threshold=0.5) -> Tuple[List[float], List[Tuple[int, int, int, int]]]:
```
with the appropriate imports, of course